lib/types/youtube.js

```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```

Comment:
@kesla I'm talking about the frame you see on youtube.com, frame is still 16:9, the video itself can have a different size, especially old videos from the old 4:3 days.